docs/IntegrationWithExistingApps.md

```diff
@@ -618,6 +624,7 @@ public class MyReactActivity extends Activity implements DefaultHardwareBackBtnH
                 .addPackage(new MainReactPackage())
                 .setUseDeveloperSupport(BuildConfig.DEBUG)
                 .setInitialLifecycleState(LifecycleState.RESUMED)
+                //.setUseOldBridge(true) // uncomment this line if your app crashes
```

Comment:
It's related to this issue: https://github.com/facebook/react-native/issues/8701